manuscript/the-meat-start.txt

```diff
@@ -38,17 +38,15 @@ Which compiles to:
         document.querySelectorAll(‘.h1bgraph’)[0]
     );
 
-As you can see, HTML code translates to `React.createElement` calls with attributes translated into a property dictionary. The beauty of this approach is that you can use React components as if they were HTML tags and that HTML attributes can be anything.
+As you can see, HTML code translates to `React.createElement` calls with attributes translated into a property dictionary. The beauty of this approach is two-pronged: you can use React components as if they were HTML tags and  HTML attributes can be anything.
```

Comment:
`tags and  HTML attributes`

You created a double-space. I'm not sure if that matters or not when this gets compiled into a pdf.